pkg/security/apis/security/types.go

```diff
@@ -16,7 +16,9 @@ type SecurityContextConstraints struct {
 
 	// Priority influences the sort order of SCCs when evaluating which SCCs to try first for
 	// a given pod request based on access in the Users and Groups fields.  The higher the int, the
-	// higher priority.  If scores for multiple SCCs are equal they will be sorted by name.
+	// higher priority. "nil" is considered a 0 priority. If scores for multiple SCCs are equal they
```

Comment:
i guess "nil" is not what you use when you want to set this via api (iow. you don't do `"priority": "nil"`. I would use the word "not set" instead of "nil".